R/utilities-goodness-of-fit.R

```diff
@@ -25,9 +25,9 @@ plotMeanGoodnessOfFit <- function(structureSet,
     observations$metaData <- readObservedDataFile(structureSet$simulationSet$observedMetaDataFile)
     observations$filter <- structureSet$simulationSet$dataFilter
 
-    # TO DO: Agree on dictionary to replace "matlabID"
-    timeColumn <- as.character(observations$metaData[observations$metaData[, "matlabID"] == "time", "nonmenColumn"])
-    dvColumn <- as.character(observations$metaData[observations$metaData[, "matlabID"] == "dv", "nonmenColumn"])
+    # TO DO: Agree on dictionary to replace "ID"

```

Comment:
is it still a TODO or can we remove the comment?

Reply:
My bad, I forgot to remove them